src/coinbase/tests/wallet_test.ts

```diff
@@ -519,6 +518,31 @@ describe("Wallet Class", () => {
     });
   });
 
+  describe("should change the network ID", () => {
+    let wallet;
+    beforeAll(async () => {
+      Coinbase.apiClients.wallet = walletsApiMock;
+      Coinbase.apiClients.address = addressesApiMock;
+      Coinbase.apiClients.wallet!.createWallet = mockReturnValue({
+        ...VALID_WALLET_MODEL,
+        network_id: Coinbase.networks.BaseMainnet,
+        server_signer_status: ServerSignerStatus.PENDING,
+      });
+      Coinbase.apiClients.wallet!.getWallet = mockReturnValue({
+        ...VALID_WALLET_MODEL,
+        network_id: Coinbase.networks.BaseMainnet,
+        server_signer_status: ServerSignerStatus.ACTIVE,
+      });
+      Coinbase.apiClients.address!.createAddress = mockReturnValue(newAddressModel(walletId));
+      wallet = await Wallet.create({
```

Comment:
Should this be a test-case under the `.create` describe block?

We're testing that we can create wallets across different networks?

Reply:
It feels like we should have a 

```
describe('.create', () => {
  let wallet;
  let networkId;

  beforeEach(async () => {
    // Shared setup
  });

  describe('when no network is specified', () => {
    beforeAll(() => networkId = Coinbase.networks.BaseSepolia);

     // it creates a wallet scoped to the default network
  });

  describe('when a network is specified', () => {
      beforeAll(() => networkId = Coinbase.networks.BaseMainnet);

      // it creates a wallet scoped to the specified network
  });
})
```